packages/node_modules/@node-red/editor-client/src/js/ui/view.js

```diff
@@ -3035,7 +3078,7 @@ RED.view = (function() {
                                         port.attr("transform", function(d) { return "translate("+x+","+((y+13*i)-5)+")";});
                                 });
                             }
-                            thisNode.selectAll("text.red-ui-flow-node-label").text(function(d,i){
+                            thisNode.selectAll("text.red-ui-flow-node-label").html(function(d,i){
```

Comment:
This looks potentially problematic -  would it let through other html in a label - or worse a script injection ?

Reply:
Correct - we try to avoid using `html()` as it can be a cause of XSS security issues. We should not be using it here. I realise the implementation works by dynamically adding `<tspan>` elements, but that should not be added like this.